compose/ui/ui-uikit/src/nativeInterop/cinterop/utils.def

```diff
@@ -1,3 +1,5 @@
 linkerOpts = -framework UIKit
 package = androidx.compose.ui.uikit.utils
-language = Objective-C
\ No newline at end of file
+language = Objective-C
+compilerOpts = -D_Float16=short
+headerFilter = CMP*
```

Comment:
All headers should be exported from the `CMP` library. I would omit this line for now.